bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BulkMutationVeneerApi.java

```diff
@@ -34,8 +37,17 @@ public class BulkMutationVeneerApi implements BulkMutationWrapper {
       BigtableClientMetrics.meter(MetricLevel.Info, "bulk-mutator.mutations.added");
   private final Batcher<RowMutationEntry, Void> bulkMutateBatcher;
 
+  private final long closeTimeoutSeconds; // If set to non-positive, never time out.
```

Comment:
Maybe move this comment to the constant definition

Reply:
I think we should make this > 0 = set timeout, 0 = disabled, < 0 is not allowed. And set a precondition check in BigtableHbaseVeneerSettings.